edx_event_bus_kafka/internal/tests/test_consumer.py

```diff
@@ -591,6 +593,26 @@ def raise_exception(*args, **kwargs):
 
         mock_consumer.commit.assert_not_called()
 
+    def test_bad_headers(self):
+        """
+        Check that if we cannot process the message headers, we raise an UnusableMessageError
+
+        The various kinds of bad headers are more fully tested in test_utils
+        """
+        msg = copy.copy(self.normal_message)
```

Comment:
I don't think you need to do this copy -- the message is recreated in setup for each test method.